### ISSUE TITLE
Wiki-Tutorial for Custom OpenGL Drawing

### ISSUE THREAD

**Issue by default0** [buggins/dlangui#257]
The APIs and steps required to do custom OpenGL drawing are hard to follow I find.

Examples are good but having a tutorial that shows how to set something up and explains steps along the way would help a great deal with using the functionality effectively.

**Comment by ifloat** [comment]
The OpenGL example is fairly easy to follow. 

w.childById("glView").backgroundDrawable = DrawableRef(new OpenGLDrawable(&doDraw));

Now just render as you would with any other OpenGL implementation in doDraw. What are you finding hard to follow?

**Comment by default0** [comment]
There is a bunch of helper APIs in dlangui.graphics.scene that I am not clear on how or when to use, as well as being unclear on how to control refresh rates of your view and things like that.
I'm sure all of these are possible in some way, but it seems really daunting to have to dig and experiment a lot to really figure them out.

Then there is that business with the new / old drawing APIs that I'm unclear about and that the example uses.

**Comment by ifloat** [comment]
I suppose this would be a good idea. I've also come to find that normal glVertex calls do nothing.
